manifests/0000_26_cloud-controller-manager-operator_01_images.configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
Do we also need an image-references as we do in MAO to get the images substituted correctly? https://github.com/openshift/machine-api-operator/blob/master/install/image-references

Reply:
Yeah, right. I added these lines in the file :+1:

Reply:
there is an image-reference below, is that not sufficient?

Reply:
It is sufficient, but it wasn't there when I reviewed ;)